Formula/mint.rb

```diff
@@ -0,0 +1,17 @@
+class Mint < Formula
+  desc "Dependency manager that installs and runs Swift command-line tool packages"
+  homepage "https://github.com/yonaskolb/Mint"
+  url "https://github.com/yonaskolb/Mint/archive/0.7.1.tar.gz"
+  sha256 "986dd967b818ee044ed0568135db8be2083ad6c3995fe6281ada9b42b224a2ec"
+  head "https://github.com/yonaskolb/Mint.git"
+
+  depends_on :xcode
```

Comment:
Looks to me like this requires Xcode 9 (or later), but only at build time. If so, the correct idiom is:

`depends_on :xcode => ["9.0", :build]`